### ISSUE TITLE
Launcher starts wrong version 1.4.28 instead of 1.4.29

### ISSUE THREAD

**Issue by kodemage** [magefree/mage#4814]
![screenshot 135](https://user-images.githubusercontent.com/7217220/39027632-71b39478-4418-11e8-8605-4d05c3f2f2f0.jpg)

despite having the latest version installed the launcher keeps launching the previous version of the client, which is not even installed. 

This is a fresh install, downloaded all new 1.4.29 through the launcher. Launcher confirms correct version is installed. Then starts old version, which does not exist but the program seems to think it's the old version

**Comment by spjspj** [comment]
You have the wrong version installed.  Use procmon or strace to work out your mistake.

**Comment by JayDi85** [comment]
Update java OS version to 1.8, wait update server to fixes or use that [full version](https://yadi.sk/d/oLKb_rzG3PwE7n) (download, unzip, run launcher and press update to inner java download).

**Comment by kodemage** [comment]
@spjspj 

You are incorrect. See screen shot. It clearly says I have the right version installed. 

If I don't then there is a bug in the launcher's update feature and in the interface which tells you what version you have installed. 

I didn't make any mistakes, and you don't have to be a jerk about this, I'm coming to you for help and to let you know that there is a bug in your software. 

I simply used the launcher's update button. Actually, I'm pretty sure I responded yes to your prompt asking if I should update to the new version. 

So, if I have the wrong version installed some how something in your launcher is installing it incorrectly. And it's doing it every time I download the client/launcher and do a fresh install. 

Let me say that again, this is repeatable after I literally delete the entire folder and download the launcher from xmage.de and let it install the client again. I cannot possibly be doing something wrong, it's literally like 3 clicks.
